src/chi/components/tables/tables.scss

```diff
@@ -2,8 +2,233 @@
 @import 'variables';
 
 .chi {
-  section {
-    &.a-table {
+
+  th {
+    text-align: left;
+  }
+
+  .a-table {
+    padding-top: 3rem;
+    position: relative;
+    border-spacing: 0;
+    font-size: $text;
+    line-height: $line-height-smaller;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+
+    table {
+      border-spacing: 0;
+      font-size: $text;
+      line-height: $line-height-smaller;
+      margin: 0;
+      padding: 0;
+      width: 100%;
+    }
+
+    tr {
+      background-color: set-color(white);
+      height: 3rem;
+    }
+
+    th {
+      border-bottom: 0.0625rem solid set-color(grey, 30);
+      font-weight: 600;
+      height: 0;
+      padding: 1rem;
+      text-transform: capitalize;
+
+      &.-sorted {
+        div {
+          align-items: center;
+          cursor: pointer;
+          display: flex;
+          flex-direction: row;
+
+          &::after {
```

Comment:
@davidojedacl aren't we using a-icon icon-chevron-up -xs?

Reply:
@davidojedacl @jllr sorting icon must be applied as a background-image like it was previously, not an inline icon